shell/components/arc-tools/handle-explorer.js

```diff
@@ -25,26 +31,26 @@ const template = Xen.Template.createTemplate(
   <div style="padding: 8px;">{{arcHandles}}</div>
 
   <div banner>Context Handles</div>
-  <div style="padding: 8px;">{{contextHandles}}</div>`
-);
+  <div style="padding: 8px;">{{contextHandles}}</div>
+`;
 
-const handleTemplate = Xen.Template.createTemplate(
-  `<!--<div style="margin-bottom: 8px;">
+const handleTemplate = html`
+  <!--<div style="margin-bottom: 8px;">
     <span>{{name}}</span>
     <a href="{{href}}" target="_blank"><i class="material-icons" style="font-size: 0.8em; vertical-align: middle;">open_in_new</i></a>
   </div>-->
   <data-explorer style="font-size: 0.8em;" object="{{data}}"></data-explorer>
-  <br>`
-);
+  <br>
+`;
 
 class HandleExplorer extends Xen.Base {
   static get observedAttributes() { return ['arc']; }
   get template() {
     return template;
   }
-  get host() {
-    return this;
-  }
+  // get host() {
```

Comment:
Background?

Reply:
Will remove the dead code. 

Xen.Base creates a shadowRoot as the template host by default. One can override and return `this` to use the element's light-dom as the host. 

This change allows handle-explorer to use shadow-dom (which is generally a good idea).